CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.27.0
+
+* Adds an overloaded `map.set()` function:
+  * `set($map, $key, $value)`: Adds to or updates $map with the specified $key
+     and $value.
+  * `set($map, $args...)`: Allows adding to or updating a map that is nested
```

Comment:
Nit: Use active voice ("Adds to or updates" rather than "Allows adding to or updating").